cpusupport/Build/cpusupport.sh

```diff
@@ -47,3 +47,4 @@ feature X86 CPUID ""
 feature X86 SSE2 "" "-msse2" "-msse2 -Wno-cast-align"
 feature X86 AESNI "" "-maes" "-maes -Wno-cast-align" "-maes -Wno-missing-prototypes -Wno-cast-qual"
 feature X86 RDRAND "" "-mrdrnd"
```

Comment:
Hmm, now that this list is getting nontrivial, maybe this and the list in `cpusupport.h` should be sorted?